extensions/amp-story/1.0/amp-story-open-page-attachment.js

```diff
@@ -108,6 +133,67 @@ const renderDefaultPageAttachmentUI = (pageEl, attachmentEl) => {
   return openAttachmentEl;
 };
 
+/**
+ * Renders inline page attachment UI.
+ * @param {!Window} win
+ * @param {!Element} pageEl
+ * @param {!Element} attachmentEl
+ * @param {!Element} attachmentHref
+ * @return {!Element}
+ */
+const renderOutlinkPageAttachmentUI = (
+  win,
+  pageEl,
+  attachmentEl,
+  attachmentHref
+) => {
+  const openAttachmentEl = buildOpenOutlinkAttachmentElement(pageEl);
+
+  // Copy href to the element so it can be previewed on hover and long press.
+  if (attachmentHref) {
+    openAttachmentEl.setAttribute('href', attachmentHref);
+  }
+
+  // Setting theme
+  openAttachmentEl.setAttribute('theme', attachmentEl.getAttribute('theme'));
+
+  // Getting elements
+  const {chipEl, ctaLabelEl} = htmlRefs(openAttachmentEl);
+
+  // Appending text & aria-label.
+  const openLabelAttr = attachmentEl.getAttribute('data-cta-text');
+  const openLabel = openLabelAttr
+    ? openLabelAttr.trim()
+    : getLocalizationService(pageEl).getLocalizedString(
+        LocalizedStringId.AMP_STORY_PAGE_ATTACHMENT_OPEN_LABEL
+      );
+  ctaLabelEl.textContent = openLabel;
+  openAttachmentEl.setAttribute('aria-label', openLabel);
+
+  // Adding image.
+  const openImgAttr = attachmentEl.getAttribute('cta-image');
+
+  if (openImgAttr === 'none') {
+    chipEl.classList.add('i-amphtml-story-outlink-page-attachment-outlink-chip-no-image');
+    ctaLabelEl.classList.add('i-amphtml-story-outlink-page-attachment-label-no-image');
+  } else {
+    const ctaImgEl = win.document.createElement('div');
+    ctaImgEl.classList.add('i-amphtml-story-outlink-page-attachment-img');
+    if (!openImgAttr) {
+      ctaImgEl.classList.add(
+        'i-amphtml-story-outlink-page-attachment-default-img'
```

Comment:
The default image class only sets the backgound image (which gets replaced anyways), so the ng-img rule for this class could be the default, and no need to have a new class attached to it